src/lib/logger.ts

```diff
@@ -0,0 +1,140 @@
+import uuid = require("uuid");
```

Comment:
Elsewhere we've generally used:
```typescript
 import * as uuid from "uuid/v4"; 
```
and then the calling code can just use `uuid()`.